ci/prow/e2e_prebuilt_kernel_module

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+echo "Label the node so that it gets the module..."
+kubectl label node minikube wants-module=kmm_ci_a
+
+echo "Build and deploy the controller image..."
+./ci/github/build_controller
+
+echo "Build and deploy the DriverContainer..." 
+./ci/github/build_driver_container
+
+echo "Deploy KMMO..."
+make deploy KUSTOMIZE_CONFIG_DEFAULT=ci/install-ci
```

Comment:
How about we use the default tags when building the images and just use `make deploy`? If the `imagePullPolicy` is `ifNotPresent` then we should use the correct image. 

WDYT?

Reply:
I think that's worth trying but maybe in a subsequent PR?

Reply:
Sure, I don't mind in which PR it is. Just wish that it will be as close as possible to what a contributor is used to run locally to make the tests easier to understand.